File: DSA1/Merge Sorted Array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        int pos = m+n-1;
        m -= 1;
        n -= 1;
        while (n >= 0) {
            nums1[pos--] = (m >= 0 && nums1[m] > nums2[n]) ? nums1[m--] : nums2[n--];
        }
    }
};

File: DSA1/Merge Sorted Array.py
class Solution:
    def merge(self, nums1: List[int], m: int, nums2: List[int], n: int) -> None:
        """
        Do not return anything, modify nums1 in-place instead.
        """
        nums3 = []
        nums4 = []
        for i in range(m):
            nums3.append(nums1[i])
        for i in range(n):
            nums4.append(nums2[i])
        ans = nums3 + nums4
        nums1.clear()
        for i in ans:
            nums1.append(i)
        nums1.sort()
